source/common/grpc/BUILD

```diff
@@ -62,7 +76,10 @@ envoy_cc_library(
     name = "common_lib",
     srcs = ["common.cc"],
     hdrs = ["common.h"],
-    external_deps = ["abseil_optional"],
+    external_deps = [
+        "abseil_optional",
+        "grpc",
```

Comment:
Do we need to take a Google gRPC dependency here?

Reply:
removed.